adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java

```diff
@@ -780,18 +781,79 @@ public void testResiliencyTokenReturnExtendedLifetimeOnwithoutRetry() throws Pac
             cacheStore.removeAll();
         }
     }
-    
+
+    @SmallTest
+    public void testVerifyManifestPermissionMissingGetAccountsPermission() throws InterruptedException, PackageManager.NameNotFoundException {
+        final FileMockContext mockContext = createMockContext();
+        when(mockContext.getPackageManager().checkPermission(Mockito.refEq("android.permission.GET_ACCOUNTS"),
+                Mockito.anyString())).thenReturn(PackageManager.PERMISSION_DENIED);
+        AuthenticationSettings.INSTANCE.setUseBroker(true);
```

Comment:
In the tearDown, you should set it to the default value to clear the test state.